### PR TITLE
fix PPO bug; add more benchmark result

### PR DESCRIPTION
**Pull request by zenghsh3**
- fix PPO bug (calculate discount reward / GAE among all episodes.)
- add more benchmark result

### CODE REVIEW

File: examples/PPO/train.py [hunk 1]
```diff
@@ -123,33 +149,25 @@ def main():
...
-        advantages = (advantages - advantages.mean()) / (
-            advantages.std() + 1e-6)
-        advantages = advantages.astype('float32')
+        train_obs, train_actions, train_advantages, train_discount_sum_rewards = build_train_data(
+            trajectories, agent)
 
-        policy_loss, kl = agent.policy_learn(obs, actions, advantages)
-        value_loss = agent.value_learn(obs, discount_sum_rewards)
+        policy_loss, kl = agent.policy_learn(
+            np.concatenate(train_obs), np.concatenate(train_actions),
```
Comment (TomorrowIsAnOtherDay) [inline examples/PPO/train.py]: It would be better to move data processing operations such as `concatenate` inside the function `build_train_data`. Also, you have concatenated the same array for several times, e.g, `train_obs`, these repeated operations are a wasteful use of computation.

**Review by TomorrowIsAnOtherDay — CHANGES_REQUESTED** [review]

**Review by TomorrowIsAnOtherDay — APPROVED** [review]
